src/app/series/[id]/editions/[editionID]/page.js

```diff
@@ -39,11 +39,13 @@ export default async function Edition({ params, searchParams }) {
     }
 
     let unpublishedVersion = false;
-    versions.items.forEach(item => {
-        if (item.state != "published") {
-            unpublishedVersion = true;
-        }
-    });
+    if (versions?.items) {
+        versions.items.forEach(item => {
+            if (item.state != "published") {
+                unpublishedVersion = true;
+            }
+        });
+    }
```

Comment:
```suggestion
    versions?.items?.forEach(item => {
        if (item.state != "published") {
            unpublishedVersion = true;
        }
    });
```
you can actually do these inside a pipe.

Reply:
Didn't know this but it's neat